src/torchmetrics/functional/audio/_deprecated.py

```diff
@@ -0,0 +1,75 @@
+from typing import Any, Callable, Optional, Tuple
+
+from torch import Tensor
+from typing_extensions import Literal
+
+from torchmetrics.functional.audio.pit import permutation_invariant_training, pit_permutate
+from torchmetrics.functional.audio.sdr import scale_invariant_signal_distortion_ratio, signal_distortion_ratio
+from torchmetrics.functional.audio.snr import scale_invariant_signal_noise_ratio, signal_noise_ratio
+from torchmetrics.utilities import _future_warning
+
+
+def _permutation_invariant_training(
+    preds: Tensor, target: Tensor, metric_func: Callable, eval_func: Literal["max", "min"] = "max", **kwargs: Any
+) -> Tuple[Tensor, Tensor]:
+    _future_warning(
+        "Importing `permutation_invariant_training` from TM functional root was deprecated and will be removed in 2.0; "
```

Comment:
Let's be more explicit: instead of saying "from TM functional root" let's say from `torchmetrics.functional`

Reply:
fixed ^^